elements/rh-table/rh-sort-button.css

```diff
@@ -0,0 +1,25 @@
+#sort-button {
+  background-color: transparent;
+  border: 0;
+}
+
+#sort-button:after {
+  content: "";
+  position: absolute;
+  inset: 0;
+  cursor: pointer;
+}
+
+#sort-button #sort-indicator {
+  color: currentcolor;
+}
+
+.visually-hidden {
```

Comment:
at this point maybe we should prepare a shared stylesheet with this. thoughts @zeroedin ?

Reply:
I think its worthwhile to have a couple shared helpers, @nikkimk and I have discussed this before as well.